packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx

```diff
@@ -121,16 +163,30 @@ export class DetailsHeader extends BaseComponent<IDetailsHeaderProps, IDetailsHe
     }
   }
 
+  public componentWillUnmount(): void {
+    if (this._subscriptionObject) {
+      this._subscriptionObject.dispose();
+      delete this._subscriptionObject;
+    }
+  }
+
   public render(): JSX.Element {
-    const { columns, ariaLabel, ariaLabelForSelectAllCheckbox, selectAllVisibility, ariaLabelForSelectionColumn } = this.props;
+    const { columns, ariaLabel, ariaLabelForSelectAllCheckbox, selectAllVisibility, ariaLabelForSelectionColumn, columnReorderOptions, onColumnClick, onColumnContextMenu } = this.props;
     const { isAllSelected, columnResizeDetails, isSizing, groupNestingDepth, isAllCollapsed } = this.state;
-
     const showCheckbox = selectAllVisibility !== SelectAllVisibility.none;
-
     const {
       onRenderColumnHeaderTooltip = this._onRenderColumnHeaderTooltip
     } = this.props;
 
+    if (!this._dragDropHelper) {

```

Comment:
>if (!this._dragDropHelper) [](start = 4, length = 26)

if (!this._dragDropHelper && this.props.columenReorderOptions) {
this._dragDropHelper = new DragDropHelper();
}